src/main/java/frc/robot/subsystems/VisionControl.java

```diff
@@ -6,19 +6,26 @@
 
 @SuppressWarnings("unused")
 public class VisionControl {
+    public enum autoState {
```

Comment:
Auto Code really needs to be in a separate PR so that we can finish and merge this

Reply:
yep